cargo-process.el

```diff
@@ -212,7 +216,9 @@ Always set to nil if cargo-process--enable-rust-backtrace is nil"
   (let* ((buffer (concat "*Cargo " name "*"))
          (path cargo-process--custom-path-to-bin)
          (path (and path (file-name-as-directory path)))
-         (command (cargo-process--maybe-read-command (concat path command)))
+         (flags (apply #'concatenate 'string cargo-process--command-flags))
```

Comment:
This will cause issues in the case of having multiple flags, since there is no space added between concatenation.

Maybe instead of a list, simply use a string for all flags?